examples/ram/resourceshareaccepter.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: ram.aws.upbound.io/v1beta1
+kind: ResourceShareAccepter
+metadata:
+  annotations:
+    meta.upbound.io/example-id: ram/v1beta1/resourceshareaccepter
+  labels:
+    testing.upbound.io/example-name: example
+  name: receiver-accept
+spec:
+  forProvider:
+    region: us-west-1
+    shareArnSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+
+---
+apiVersion: ram.aws.upbound.io/v1beta1
+kind: PrincipalAssociation
+metadata:
+  annotations:
+    meta.upbound.io/example-id: ram/v1beta1/resourceshareaccepter
+    uptest.upbound.io/pre-delete-hook: testhooks/pre-delete-principal.sh
+  labels:
+    testing.upbound.io/example-name: example
+  name: sender-invite
+spec:
+  providerConfigRef:
+    name: peer
```

Comment:
Thank you @turkenf for your dedication in improving our test automation and going the extra mile to keep our resources uptestable, even if they are cross-account resources.